From 391bec0d0fce9a1d763d8318a5f1732ea51cc7cc Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Wed, 25 Feb 2009 18:20:24 -0800
Subject: Staging: slicoss: remove unused #defines

There are a number of "config" defines that do nothing, remove them.

Cc: Lior Dotan <liodot@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/slicoss/slicbuild.h |   32 ++++++++++++--------------------
 drivers/staging/slicoss/slicoss.c   |    2 --
 2 files changed, 12 insertions(+), 22 deletions(-)

--- a/drivers/staging/slicoss/slicbuild.h
+++ b/drivers/staging/slicoss/slicbuild.h
@@ -43,52 +43,44 @@
 #ifndef _SLIC_BUILD_H_
 #define _SLIC_BUILD_H_
 
-#ifndef SLIC_PRODUCTION_BUILD
-#define SLIC_PRODUCTION_BUILD			1
-#endif
-#ifndef SLIC_FAILURE_RESET
-#define SLIC_FAILURE_RESET			1
-#endif
-#define DBG                         1
+#define DBG					1
+
 #ifndef SLIC_ASSERT_ENABLED
 #define SLIC_ASSERT_ENABLED			1
 #endif
-#ifndef SLIC_MCAST_ENABLED
-#define SLIC_MCAST_ENABLED			1
-#endif
+
 #ifndef SLIC_GET_STATS_ENABLED
 #define SLIC_GET_STATS_ENABLED			1
 #endif
+
 #ifndef SLIC_GET_STATS_TIMER_ENABLED
 #define SLIC_GET_STATS_TIMER_ENABLED		0
 #endif
+
 #ifndef SLIC_PING_TIMER_ENABLED
-#define SLIC_PING_TIMER_ENABLED		1
-#endif
-#ifndef SLIC_IOCTL_SUPPORT_ENABLED
-#define SLIC_IOCTL_SUPPORT_ENABLED		1
-#endif
-#ifndef ATK_DEBUG
-#define ATK_DEBUG				1
+#define SLIC_PING_TIMER_ENABLED			1
 #endif
+
 #ifndef SLIC_POWER_MANAGEMENT_ENABLED
 #define SLIC_POWER_MANAGEMENT_ENABLED		0
 #endif
+
 #ifndef SLIC_INTERRUPT_PROCESS_LIMIT
 #define SLIC_INTERRUPT_PROCESS_LIMIT		1
 #endif
+
 #ifndef LINUX_FREES_ADAPTER_RESOURCES
 #define LINUX_FREES_ADAPTER_RESOURCES		1
 #endif
+
 #ifndef SLIC_OFFLOAD_IP_CHECKSUM
 #define SLIC_OFFLOAD_IP_CHECKSUM		1
 #endif
-#ifndef SLIC_POWER_MANAGEMENT_ENABLED
-#define SLIC_POWER_MANAGEMENT_ENABLED		0
-#endif
+
 #ifndef STATS_TIMER_INTERVAL
 #define STATS_TIMER_INTERVAL			2
 #endif
+
 #ifndef PING_TIMER_INTERVAL
 #define PING_TIMER_INTERVAL			1
 #endif
--- a/drivers/staging/slicoss/slicoss.c
+++ b/drivers/staging/slicoss/slicoss.c
@@ -58,8 +58,6 @@
 #define SLIC_DUMP_ENABLED               0
 #define KLUDGE_FOR_4GB_BOUNDARY         1
 #define DEBUG_MICROCODE                 1
-#define SLIC_PRODUCTION_BUILD	        1
-#define SLIC_FAILURE_RESET	            1
 #define DBG                             1
 #define SLIC_ASSERT_ENABLED		        1
 #define SLIC_GET_STATS_ENABLED			1
